pkg/network/node/networkpolicy.go

```diff
@@ -30,6 +30,7 @@ import (
 )
 
 const HostNetworkNamespace = "openshift-host-network"
+const NetPolIsolationCookie = 1147582955
```

Comment:
maybe clarify that this is just a randomly-chosen value (and, eg, it's not referenced anywhere else)